Dockerfile

```diff
@@ -8,8 +8,12 @@ WORKDIR /tmp
 COPY apt.txt /tmp/apt.txt
 RUN apt-get update &&\
     apt-get install -y $(grep -vE "^\s*#" apt.txt  | tr "\n" " ") &&\
-    ln -s /usr/bin/nodejs /usr/bin/node &&\
-    pip install pip --upgrade
+    ln -s /usr/bin/nodejs /usr/bin/node
+
+# Install pip
+RUN curl --silent --location https://bootstrap.pypa.io/get-pip.py > get-pip.py &&\
```

Comment:
I feel like you'll get an error about `curl` not being installed here since I ran into it earlier, but we can wait for the tests to succeed or fail